instagram/src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
If you're not using this it's a better practice to take it out, it's leaving an error in the console.

Reply:
Delete the file as well